github/orgs_rules_test.go

```diff
@@ -18,14 +18,16 @@ func TestOrganizationsService_GetAllOrganizationRulesets(t *testing.T) {
 	t.Parallel()
 	client, mux, _ := setup(t)
 
-	mux.HandleFunc("/orgs/o/rulesets", func(w http.ResponseWriter, r *http.Request) {
+	org := "o"
```

Comment:
Throughout this PR, let's please revert this change. `org := "o"` doesn't provide any additional clarity (and only increases the number of lines) because `"o"` throughout this repo always represents an organization and `"r"` always represents the name of a repo. In fact, `enterprise := "e"` doesn't really add a whole lot of value either, but you can leave it if you want since it will be one of first representations of an enterprise.

Reply:
Sure.